scss/book-list.scss

```diff
@@ -0,0 +1,46 @@
+.container-list {
+  display: flex;
+  flex-wrap: wrap;
+  width: 960px;
+  height: 100%;
+  margin:30px 0 50px 0;
+  justify-content: space-between
+}
+
+.card {
+  background-color: $white-secondary;
+  box-shadow: 4px 4px 10px 0 $shadow;
+  display: flex;
+  flex-direction: column;
+  height: 300px;
+  margin: 36px 0 0 ;
+  width: 198px;
+ 
+  
+  card-h2 {
+    color: $black-default;
+    font-size: 16px;
+    font-weight: 700;
+    height: 15px;
+    line-height: 18px;
+    margin-left: 24px;
+  }
+
+
+  card-h3 {
+    color: $grey;
+    font-size: 14px;
+    height: 14px;
+    line-height: 18px;
```

Comment:
Did you use the invision css code in this class? The line-height is bigger than the height

Reply:
![Captura de pantalla de 2021-03-15 12-39-27](https://user-images.githubusercontent.com/56491095/111192635-0c21d880-8598-11eb-8ac0-a6960cd18a0f.png)

Reply:
Mmm, I think it's better to remove the line-height here or assign to it the same value as height, @mcallegari10 what do you think?

Reply:
Yes, it must be that height to create some space. Set it to font size and should be fine 😄